getting-started-guides/csp/databricks/init-notebook-for-rapids-spark-xgboost-on-databricks-gpu-7.0-ml.ipynb

```diff
@@ -25,7 +25,7 @@
     "%sh\n",
     "cd ../../dbfs/FileStore/jars/\n",
     "wget -O cudf-0.18.1.jar https://search.maven.org/remotecontent?filepath=ai/rapids/cudf/0.18.1/cudf-0.18.1.jar\n",
-    "wget -O rapids-4-spark_2.12-0.4.0.jar https://search.maven.org/remotecontent?filepath=com/nvidia/rapids-4-spark_2.12/0.4.0-databricks/rapids-4-spark_2.12-0.4.0.jar\n",
+    "wget -O rapids-4-spark_2.12-0.4.1.jar https://search.maven.org/remotecontent?filepath=com/nvidia/rapids-4-spark_2.12/0.4.1-databricks/rapids-4-spark_2.12-0.4.1.jar\n",
```

Comment:
https://search.maven.org/remotecontent?filepath=com/nvidia/rapids-4-spark_2.12/0.4.1-databricks/rapids-4-spark_2.12-0.4.1.jar is not a valid URL.
It should be https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/0.4.1/rapids-4-spark_2.12-0.4.1.jar.

Reply:
Done